vms/event/models.py

```diff
@@ -12,6 +12,7 @@ class Event(models.Model):
                 r'^[(A-Z)|(a-z)|(0-9)|(\s)|(\.)|(,)|(\-)|(!)|(\')]+$', ),
         ],
         )
+    description = models.TextField(blank=True)
```

Comment:
Shouldn't we keep an upper limit on it's length?

Reply:
There is no such restriction in portal, might create issues later @kamsuri

Reply:
But i think there should be an upper limit. @abhishekarya286 @ayushgarg1804 your views?